Tests/MessageBusProcessorTest.php

```diff
@@ -54,6 +55,27 @@ public function testProcess()
         $this->assertSame(Processor::ACK, $messageBusProcessor->process($message, $contextProphecy->reveal()));
     }
 
+    /**
+     * @expectedException \Symfony\Component\Messenger\Exception\MessageDecodingFailedException
```

Comment:
Please dont use annotations for this. 

Use `$this->expectException()` instead. And put that line just before you expect that exception.

Reply:
@Nyholm removed the expectedException and changed into Reject message Processor (which is the correct implementation).